fizz-buzz.test.js

```diff
@@ -1,6 +1,31 @@
 const { fizzBuzz } = require('./fizz-buzz');
 
-// 1. write a describe to contain our tests
-// 2. add a first test. You could use one of the console.log tests in the other file as a starting point
-// 3. run the test with `npm test` to see if it passes
-// 4. add another test, and another, and another, until you've tested all the functionality of the fizzBuzz function
+describe("fizzBuzz", () => {
+  it('returns the number is divisible by 3 and 5, return FizzBuzz', () => {
+    const result = fizzBuzz(15);
+    expect(result).toEqual('FizzBuzz');
+  })
+})  
+
+  describe("Fizz", () => {
```

Comment:
Usually, you'd group several tests together in a single `describe`, so no need for these extra `describe` wrappers